### PR TITLE
Include information need in documentation

### LINKED ISSUE #162: Include information need in documentation
The documentation should integrate a description of how an information need is represented (incl. target items, constraints, and requests), and its use to customise generated utterances and evaluation.

### PR DESCRIPTION
**Pull request by NoB0**
As of now the documentation page is isolated, a link to it should be integrated.

Fixes #162

### CODE REVIEW

File: docs/source/information_need.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    {
+        "constraints": {
+            "genre": "comedy"
+        },
+        "requests": ["plot"],
+        "target_items": ["Jump Street", "The Hangover"]
+    }
+
+The information need is a core element of the user simulator as it is used to customize the generated responses and can serve for the evaluation of the conversational recommender system.
```
Comment (kbalog) [inline docs/source/information_need.rst:19]: "can serve for the evaluation" => "can serve as reference for the evaluation"

**Review by kbalog — APPROVED** [review]
LGTM

File: docs/source/information_need.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Information need
+================
+
+The information need defines in a structured manner what the user is looking for. It comprises constraints, requests, and target items. The constraints specify the slot-value pairs that the item of interest must satisfy, while the requests specify the slots for which the user wants
```
[the remark this answers is not in the document]
Comment (NoB0) [reply docs/source/information_need.rst:4]: Done.

File: docs/source/information_need.rst (hunk 1 above)
Comment (NoB0) [reply docs/source/information_need.rst:19]: Done.

**Pull request merged**